cloudflare/resource_cloudflare_teams_accounts.go

```diff
@@ -86,12 +120,34 @@ func resourceCloudflareTeamsAccountUpdate(d *schema.ResourceData, meta interface
 	if ok {
 		updatedTeamsAccount.Settings.ActivityLog = &cloudflare.TeamsActivityLog{Enabled: activtyLog.(bool)}
 	}
+
+	browserIsolation, ok := d.GetOkExists("url_browser_isolation_enabled")
+	if ok {
+		updatedTeamsAccount.Settings.BrowserIsolation = &cloudflare.BrowserIsolation{UrlBrowserIsolationEnabled: browserIsolation.(bool)}
+	}
+
+	fipsTls, ok := d.GetOkExists("fips_tls")
```

Comment:
i think #1380 already covered this. you might need to rebase it in with your changes.

Reply:
done